widgets/DynamicSymbology/Widget.js

```diff
@@ -67,7 +67,7 @@ function(declare, BaseWidget, LayerInfos, dom, domConstruct, on, Map, Color, Col
 	  var curColor = new Color([92,92,92]);
 	  var dynamicSym = this;
 
-	  dynamicSymbology.isSmartMapping = false;
+	  dynamicSymbology.isSmartMapping = true;
```

Comment:
Hrm. Any examples? What I'm seeing deployed looks pretty darn good. Obviously still no choice of color ramps, is that the kind of control you'd expect to have?